hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1040,7 +1078,19 @@ long generateDiffReport(final String jobId,
                 SnapshotDiffReportOzone.getDiffReportEntry(DiffType.MODIFY,
```

Comment:
In case of two renames ending up back to same name we might end up with a modify entry, which we are ok with a rename entry (ideally that also shouldn't be there) since it is just a meta change but modify will be expensive as we will end up copying the whole data all over again.